prefect_saturn/core.py

```diff
@@ -303,6 +301,7 @@ def _get_environment(
                 adapt_kwargs=adapt_kwargs,
             ),
             job_spec_file=local_tmp_file,
+            labels=[str(urlparse(self._base_url).hostname), "saturn-cloud"],
```

Comment:
just want to note here that if this Prefect proposal to remove `KubernetesJobEnvironment` ever happens, we'll have to find a different place to put labels: https://github.com/PrefectHQ/prefect/issues/2928

but we'll have lots of other problems if that happens too so doesn't affect this PR :grinning:

Reply:
Oh fun. I kinda liked that we could have default labels on the environment, and then allow users to set their own on the flow (if we get around to user-defined labels on agents)